integration/nginx_reload_test.go

```diff
@@ -0,0 +1,111 @@
+package integration_test
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/paketo-buildpacks/occam"
+	"github.com/paketo-buildpacks/packit/v2/pexec"
+	"github.com/sclevine/spec"
+
+	. "github.com/onsi/gomega"
+	. "github.com/paketo-buildpacks/occam/matchers"
+)
+
+func testNginxReload(t *testing.T, context spec.G, it spec.S) {
+	var (
+		Expect     = NewWithT(t).Expect
+		Eventually = NewWithT(t).Eventually
+
+		pack   occam.Pack
+		docker occam.Docker
+		source string
+		name   string
+	)
+
+	it.Before(func() {
+		pack = occam.NewPack().WithVerbose()
+		docker = occam.NewDocker()
+	})
+
+	context("when BP_LIVE_RELOAD_ENABLED is true", func() {
```

Comment:
@joshuatcasey I'm confused about this. `BP_LIVE_RELOAD_ENABLED` is not set to true - does this line need to be modified? Or should `BP_LIVE_RELOAD_ENABLED` be set?

I am assuming just this line needs to change?

Reply:
@joshuatcasey this still needs to change since the env var isn't set.